### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ngoiyaeric**
rust -> comprehensive rust

### CODE REVIEW

**Review by djmitche — COMMENTED** [review]
At least for the initial 3-day course, there's no need to actually install this repo, right? So it's reasonable to assume that people installing the repo are hacking on the course itself?

I think this doesn't hurt, but I'm not sure it helps, either.

**Comment by ngoiyaeric** [comment]
> At least for the initial 3-day course, there's no need to actually install this repo, right? So it's reasonable to assume that people installing the repo are hacking on the course itself?
> 
> I think this doesn't hurt, but I'm not sure it helps, either.

I created an issue cause I was hacking on the course and was stuck on installation, so I thought that this edit would be helpful for anyone hacking on the course in the future.

File: README.md [hunk 1]
```diff
@@ -37,7 +37,15 @@ The course is built using a few tools:
 - [mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers)
 - [mdbook-exerciser](mdbook-exerciser/)
 
-First clone the repository:
+First install rust: 
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```
Comment (mgeisler) [inline README.md:43]: I don't like to replicate these instructions — there should only be one place on the Internet that needs to know about the flags-of-the-day to use with `curl` :slightly_smiling_face: That is also why we refer to rustup.sh in the [Installation](https://google.github.io/comprehensive-rust/cargo.html#installation) section.

File: README.md [hunk 2]
```diff
@@ -37,7 +37,15 @@ The course is built using a few tools:
...
+source $HOME/.cargo/env
+rustc --version
+```
+
+Clone the repository:
```
Comment (mgeisler) [inline README.md:48]: Let us simply send people to rustup.rs:

````suggestion
First install Rust by following the instructions on https://rustup.rs/. Then clone this repository:
````

**Review by mgeisler — APPROVED** [review]
Hey @ngoiyaeric, I simplified your change a lot while trying to keep the spirit of it. I hope that works for you!

File: README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ The course is built using a few tools:
 - [mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers)
 - [mdbook-exerciser](mdbook-exerciser/)
 
-First clone the repository:
+First install Rust by following the instructions on https://rustup.rs/. Then clone this repository:
```
Comment (mgeisler) [inline README.md:40]: Let me quickly fix the formatting according to `dprint`:

```suggestion
First install Rust by following the instructions on https://rustup.rs/. Then
clone this repository:
```

**Comment by mgeisler** [comment]
> Hey @ngoiyaeric, I simplified your change a lot while trying to keep the spirit of it. I hope that works for you!

You will still need to sign the CLA since you are the original author of this (modified) change.

**Pull request merged**